Frends.ServiceBus/ServiceBus.cs

```diff
@@ -341,6 +348,109 @@ private static Encoding GetEncodingFromContentType(string contentTypeString, Enc
                 }
             }
         }
+
+        static async public Task<ReadResult> SendAndWaitForResponse(SendAndWaitForResponseInput input, SendAndWaitForResponseOptions options, CancellationToken token = new CancellationToken())
+        {
+            // Prepare queues
+            if (options.CreateQueues)
+            {
+                await EnsureQueueExists(input.Queue, input.ConnectionString, false);
+                await EnsureQueueExists(input.ReplyToQueue, input.ConnectionString, true);
+            }
+
+            QueueClient requestClient = MessagingFactory.CreateFromConnectionString(input.ConnectionString).CreateQueueClient(input.Queue);
+            QueueClient responseClient = MessagingFactory.CreateFromConnectionString(input.ConnectionString).CreateQueueClient(input.ReplyToQueue);
+            MessageSession session = responseClient.AcceptMessageSession(input.SessionID);
```

Comment:
You are probably right. If I have the time today then I'll see if I can get this to work with the usual read task. Although I think that a separate task for request/response is also beneficial. It is easy for FRENDS users to grasp as it is almost the same thing as using a web request task for sending a req and getting a response. But I agree that it would also be beneficial to be able to read sessions using the Read task.